db/teams.js

```diff
@@ -100,9 +100,11 @@ function createTeam (teamName, email, password, isEligible) {
 }
 
 module.exports = {
-  createTeam,
-  getTeam,
-  getTeamByName,
-  editTeam,
-  getAllTeamNames
-}
+  createTeam: createTeam,
```

Comment:
Same comment as the other PR, you should be able to change the format from `createTeam: createTeam` to just `createTeam`